hicommonwealth/commonwealth
# Filter unactive communities

## Link to Issue
Closes: #5470

## Description of Changes
- Adds new query option to get "active" communities (as described by Zak)– those that have at least 1 topic and 10 members
- Adds migration to add `community_id` indexes to Topics and Addresses to potentially speed up query
- Adds caching to the function since the query is still somewhat slow (1.2s) and the data doesn't change frequently

## Test Plan
- Go to http://localhost:8080/communities
  - Confirm that the UI says there are 1K+ communities, but only shows a few hundred

## Deployment Plan
N/A

## Other Considerations
N/A
